packages/css-library/README.md

```diff
@@ -1,3 +1,17 @@
 # css-library
 
-This package is the early discovery work on basing VADS styles off of USWDS v3.
+## Stylesheets
+
+The stylesheets generated by this library are based on USWDS version 3 and are intended to eventually replace the Formation CSS library:
+
+- fonts.css
+- elements.css
+- utilties.css
+
+To generate stylesheets, run `yarn build:stylesheets` or `yarn build:minify`
+
+## Design Tokens
+
+The tokens generated by this library will be available in several different formats so that various tools and environments can use them.
```

Comment:
Are the token values generated by this library based on USWDS V3 tokens, similar to the stylesheets you mentioned above?

Reply:
At the moment, these are VA specific primitive tokens. We manually added a [couple of v3 colors](https://github.com/department-of-veterans-affairs/component-library/blob/5799191fe3ed92471190d00132e437fc436bb4c8/packages/css-library/tokens/color.json#L58-L65) at one point to unblock us from modifying the way error states displayed on form web components ([our error state has a left border](https://design.va.gov/storybook/?path=/docs/uswds-va-checkbox--error)).

In the near future, USWDS token colors that we want should eventually be programmatically added to our available tokens here though. We have a separate ticket for discovery how we are going to do that: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1967